### PR TITLE
Implemented detection of air mode activation in Betaflight

### PR DESCRIPTION
**Pull request by basdelfos**
Implemented detection of air mode activation in Betaflight (requires Betaflight 2.1.5 or higher).

From Boris' Betaflight thread:

**Air mode**
* Enable Air mode on switch. I personally would prefer to use it together or next to an arming switch to always get full control at 0 throttle or separate switch so you can switch between normal and Air mode. When you use stick arming you will loose ability to yaw with 0 throttle, which might be a big miss. I was a stick armer before and really had to use myself to use switch to arm with air mode, but it was worth switching! All inverted and 0 stuff is much more fun now and disarming can be done quickly.
Of course you can stick with stick arming, but realize that there is now yaw with 0 throttle so you are missing a bit of fun. 
* Motor stop disables ability to use 0 throttle as the motors would stop. So you have to ask yourself if it is not worth of switching it off so you can enjoy the full fun

Link to thread: http://www.rcgroups.com/forums/showthread.php?t=2464844

Here is some visual demonstration of how to use air mode and enjoy more in air: https://youtu.be/b0qVUa4AeDQ

### CODE REVIEW

File: MW_OSD/Config.h [hunk 1]
```diff
@@ -33,10 +33,10 @@
 // Choose ONLY ONE option from the following long list :-
 
 // latest release...
-#define MULTIWII                  // Uncomment this if you are using latest MULTIWII version from repository (2.4 at time of this MWOSD release)
+//#define MULTIWII                  // Uncomment this if you are using latest MULTIWII version from repository (2.4 at time of this MWOSD release)
 //#define BASEFLIGHT                // Uncomment this if you are using latest BASEFLIGHT version from repository (Stable 2015.06.27 at time of this MWOSD release)
 //#define TAULABS                   // Uncomment this if you are using the latest Tau Labs MSP Module
-//#define CLEANFLIGHT               // Uncomment this if you are using latest CLEANFLIGHT version from repository (1.9.0 at time or this MWOSD release)
+#define CLEANFLIGHT               // Uncomment this if you are using latest CLEANFLIGHT version from repository (1.9.0 at time or this MWOSD release)
```
Comment (tracernz) [inline MW_OSD/Config.h]: All changes to this file are unrelated changes to defaults?

**Comment by tracernz** [comment]
The "air mode" name is very unfortunate as it is completely non-descriptive. -_-

**Comment by tracernz** [comment]
Also, any reason for deleting fontL.h and fontD.h?

File: MW_OSD/Config.h (hunk 1 above)
Comment (basdelfos) [inline MW_OSD/Config.h]: Hmm, yes. These changes are unrelated to the airmode implementation, they are changes to default for my own use. Should perhaps be left out while merging with original branche.

**Comment by basdelfos** [comment]
@tracernz - what do you expect to be more descriptive about the "air mode" name?
I didn't delete the fontL.h and fontD.h files, they are only refreshed with new byte array data, because I also changed the .mcm font files to add the symbols for displaying airmode.

**Comment by tracernz** [comment]
@basdelfos It's not a problem with your PR, just that "air mode" offers no clue as to what the feature/mode actually does which is unfortunate. ;-) It's a shame borisb didn't pick something better.

~~The font headers look like all the content in the arrays is deleted when viewing on Github?~~ Weird, I can see some content now, couldn't earlier when viewing with another browser.
